docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -1,86 +1,94 @@
 .. _release_and_maintenance:
 
-Release and maintenance
-=======================
+************************
+Releases and maintenance
+************************
 
-This section describes the Ansible and ``ansible-core`` releases.
-Ansible is the package that most users install. ``ansible-core`` is
-primarily for developers.
+This section describes release cycles, rules, and maintenance schedules for both Ansible community projects: the Ansible community package and ``ansible-core``. The two projects have different versioning systems, maintenance structures, contents, and workflows.
+
+====================================================  =================================================
+Ansible community package                             ansible-core
+====================================================  =================================================
+each major release is a new major version (4.0.0)     continues "classic Ansible" versioning (2.11)
+follows semantic versioning rules                     does not use semantic versioning
+maintains only one version at a time                  maintains latest version plus two older versions
+includes language, runtime, and selected Collections  includes only the language and runtime
+developed and maintained in Collection repositories   developed and maintained in ansible/ansible repo
+====================================================  =================================================
+
+Most community users install the Ansible community package. The ``ansible-core`` option is primarily for developers.
```

Comment:
```suggestion
Most community users install the Ansible community package. The ``ansible-core`` option is primarily for developers and users who want to install only collections they need.
```
Or something like that. I think to install `ansible-core` + collections that and only that needed from Galaxy is a very good approach for some users, IMO